index.html

```diff
@@ -326,14 +333,16 @@
           $('#info').html('');
           // make sure initally displayed selection matches start value
           d3.select('#pointsets').property('value', pointset);
-    }).addTo(map);
+      }).on('movedlocation', function(pt) {
+          moveHighlightedBlock(pt);
+      }).addTo(map);
 
     var isochroneShowing = false;
     $('#block-info').on('click', '#toggle-travelshed', function(e) {
         e.preventDefault();
         if (!isochroneShowing) {
-          var loc = L.latLng(e.target.dataset.lat, e.target.dataset.lng);
-          otpaLayer.setLocation(loc);
+          //var loc = L.latLng(e.target.dataset.lat, e.target.dataset.lng);
+          //otpaLayer.setLocation(loc);
```

Comment:
No, it's being set in the polygon click event handler now. I'll go ahead and remove these commented lines. (Wasn't sure what was in `e.target.dataset`.)